CHANGELOG_iOS.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [1.65.122](https://github.com/brave/brave-browser/releases/tag/v1.65.122)
+
+### Web3
+
+ - Fixed “Allow Biometrics Toggle” not being updated in Wallet settings. ([#36669](https://github.com/brave/brave-browser/issues/36669))
+
+### General
+
+ - Added support for Dutch language. ([#36939](https://github.com/brave/brave-browser/issues/36939))
+ - Added support for marketplace URI scheme. ([#37650](https://github.com/brave/brave-browser/issues/37650))
+ - Updated search result attribution to be more unified. ([#33469](https://github.com/brave/brave-browser/issues/33469))
+ - Fixed crash related to opening and closing many tabs right away. ([#37224](https://github.com/brave/brave-browser/issues/37224))
```

Comment:
Maybe we can use `quickly` rather than `right away`? For example:

```
- Fixed crash related to quickly opening and closing many tabs. ([#37224](https://github.com/brave/brave-browser/issues/37224))
```